tagstudio/src/qt/ts_qt.py

```diff
@@ -1449,12 +1521,20 @@ def update_thumbs(self):
 
         for i, item_thumb in enumerate(self.item_thumbs, start=0):
             if i < len(self.nav_frames[self.cur_frame_idx].contents):
-                filepath = ""
+                filepath: Path = None  # Initialize
                 if self.nav_frames[self.cur_frame_idx].contents[i][0] == ItemType.ENTRY:
                     entry = self.lib.get_entry(
                         self.nav_frames[self.cur_frame_idx].contents[i][1]
                     )
-                    filepath = self.lib.library_dir / entry.path / entry.filename
+                    filepath: Path = self.lib.library_dir / entry.path / entry.filename
+
+                    try:
+                        item_thumb.delete_action.triggered.disconnect()
+                    except RuntimeWarning:
+                        pass
```

Comment:
Why not log this?

Reply:
The `.disconnect()` method from Qt is currently a pain point for us, as they've recently changed it from throwing a `RuntimeException` to a `RunetimeWarning` - I'm also not sure if it catches here properly due to that. It's also something that for our purposes, we don't really care if it fails - we just want to ensure that it's disconnected without caring about the previous state, initialized or otherwise. Overall we need a better solution for this, but the recent changes in Qt have made this more difficult to come up with a proper solution for.